Project4/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+<head>
+    <title>Importance of being a generalists</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="first" href="first.js">
+</head>
+<div class="site-wrapper">
+<body style="background-color:#250050">
+    <link href="https://fonts.googleapis.com/css?family=Abel|Arimo|Arvo|Cabin|Fjalla+One|Kanit|Nunito|Oswald|PT+Sans|PT+Sans+Narrow|Poppins|Rubik|Titillium+Web" rel="stylesheet">
+    <a name="home"></a>
+    <h1><b>SPECIALIZIED VERSITILITY</b></h1>
```

Comment:
A few things here:
1. Your title has a typo in it
1. You don't need an `h1` and `b` both -- the `b` element is used to bring attention to an element or make things bold -- both of these things are already accomplished with an `h1` element.
1. Your markup should avoid all caps text and that should be controlled via CSS `text-transform: uppercase;` instead:
```suggestion
    <h1>Specialized Versatility</h1>
```